docs/CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ Entries inside each section should be ordered by type:
 
 # unreleased - YYYY-MM-DD
 ## Python API
+* [Added] New 'unversioned' parameter to `Package.set_dir()` and `Package.set()` for use with HealthOmics and other S3 URIs that lack permissions for `ListBucketVersions` and/or `GetObjectVersion` ([#3927](https://github.com/quiltdata/quilt/pull/3927))
```

Comment:
I'm not sure that S3 URIs that lack permissions is semantically correct

Reply:
Any suggestions? The term "buckets that lack sufficient permissions" feels even less accurate, since Omics has S3 URIs but no actual "buckets" :-(

Reply:
I mean that it's user lacks permissions to "S3 URIs"